src/app/pages/candidate-details/candidate-details.page.html

```diff
@@ -1,8 +1,8 @@
 <ion-header>
     <ion-toolbar mode="ios">
-        <ion-title class="ion-text-center">Test details - {{ pageState.name }}</ion-title>
+        <ion-title id="title" class="ion-text-center">Test details - {{ pageState.name }}</ion-title>
```

Comment:
Can you make the ID a bit more bespoke i.e. instead of "title" have "test-details-title"

Reply:
Change to reflect commit